src/couch_replicator/test/eunit/couch_replicator_test_helper.erl

```diff
@@ -206,6 +206,7 @@ test_setup() ->
     Hashed = couch_passwords:hash_admin_password(?PASSWORD),
     Persist = false,
     ok = config:set("admins", ?USERNAME, ?b2l(Hashed), Persist),
+    ok = config:set("replicator", "use_bulk_get", "true", Persist),
```

Comment:
use_bulk_get should already default to true or is some test leaving it disabled? in that case we should fix that test

It also doesn't seem to be related to changes_sync PR in general? That's fine that it's in a separate commit but the PR message should mention both or indicate why we need it.

Reply:
If we had set different settings in the default.ini file, these tests would fail.
So just trying to set it to `true` explicitly in tests.

Reply:
> and don't reset it back ...

Good find, Ronny! That's a good thing to fix. However if we still set it with Persist=false it should be wiped out on next application cycle restart.